### PR TITLE
Pull Request for Issue2232: BioXAS mirror refactor - removing mask.

### PR DESCRIPTION
**Pull request by helfrij**
Changes:
- Renamed CLSMirrorMaskState back to BioXASMirrorMaskState.
- Removed BioXASM1MirrorMask class.
- Removed BioXASMirrorMaskView class.

### CODE REVIEW

**Comment by helfrij** [comment]
Hey @XuetaoLiu-CLS, would you review?

File: source/beamline/BioXAS/BioXASMirrorMaskState.cpp [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	if (setBaseControl(newControl))
+		emit upperBladeChanged(newControl);
+}
+
+AMAction3* BioXASMirrorMaskState::createMoveAction(double setpoint)
+{
+	AMAction3 *result = 0;
+
+	if (int(setpoint) == Closed)
+		result = createMoveToClosedAction();
```
Comment (XuetaoLiu-CLS) [inline source/beamline/BioXAS/BioXASMirrorMaskState.cpp]: don't we need a moveToOpenAction() ?

**Comment by XuetaoLiu-CLS** [comment]
Looks good other than a question.

Time Estimate: 10 minutes

File: source/beamline/BioXAS/BioXASMirrorMaskState.cpp (hunk 1 above)
Comment (helfrij) [inline source/beamline/BioXAS/BioXASMirrorMaskState.cpp]: No, because we don't have setpoints for where the mask position should be yet.

**Comment by dretrex** [comment]
Looks good.
Time Estimate: 5 minutes

**Pull request merged**